.eslintrc.js

```diff
@@ -2,13 +2,27 @@ module.exports = {
   root: true,
   extends: [
     'airbnb-base',
+    'plugin:jest/all',
   ],
   plugins: [],
   env: {
     node: true,
   },
   rules: {
     'implicit-arrow-linebreak': 0,
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: [
+          'scripts/*.js',
+          'test/**/*.js'
+        ]
+      },
+    ],
+    'jest/no-hooks': ['error', {
+      allow: ['beforeEach'],
```

Comment:
Needs to be deactivated to prevent authentication context issues in some test files.